test/e2e/condition.py

```diff
@@ -0,0 +1,146 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""Utility functions to help processing Kubernetes resource conditions"""
+
+# TODO(jaypipes): Move these functions to acktest library. The reason these are
+# here is because the existing k8s.assert_condition_state_message doesn't
+# actually assert anything. It returns true or false and logs messages.
```

Comment:
@RedbackThomson see this comment? :)

Reply:
Why is this a blocker for these methods to be moved into acktest though?

Reply:
it's not a blocker to move into acktest.

Reply:
https://github.com/aws-controllers-k8s/test-infra/pull/166

Reply:
Thanks! Merged!